templates/blog/article.html

```diff
@@ -1,13 +1,17 @@
-{% extends_with_args "templates/one-column.html" with title=article.title.rendered meta_description=article.excerpt.raw  meta_image=article.image.source_url %}
+{% extends "templates/one-column.html" %}
+
+{% block title %}article.title.rendered{% endblock %}
+{% block meta_description %}article.excerpt.raw {% endblock %}
+{% block meta_image %}article.image.source_url{% endblock %}
```

Comment:
These values need to be wrapped in curly braces, at the moment they're being evaluated as strings.